packages/xo-server/src/fatfs-buffer.js

```diff
@@ -47,7 +47,7 @@ export function init() {
       Reserved1: 0,
       BootSig: 41,
       VolID: 895111106,
-      VolLab: 'NO NAME    ',
+      VolLab: 'cidata    ',
```

Comment:
Add a param with a default value of `'NO NAME    '` to the function.

Also, add an assert to make sure the label length is `11`.